lua/lazyvim/config/keymaps.lua

```diff
@@ -110,6 +110,9 @@ map("n", "<leader>uf", require("lazyvim.plugins.lsp.format").toggle, { desc = "T
 map("n", "<leader>us", function() Util.toggle("spell") end, { desc = "Toggle Spelling" })
 map("n", "<leader>uw", function() Util.toggle("wrap") end, { desc = "Toggle Word Wrap" })
 map("n", "<leader>ul", function() Util.toggle_number() end, { desc = "Toggle Line Numbers" })
+if Util.has("mini.pairs") then
+  map("n", "<leader>up", function() Util.toggle_autopairs() end, { desc = "Toggle Auto Pairs" })
+end
```

Comment:
Another possibility is adding this mapping and toggle function to the `mini.pairs` plugin spec, e.g. in a `config` function. It would remove the need for the plugin enabled/disabled check, but I'm not sure it's preferable/better. Just a thought.

Reply:
Keeping them in once place probably makes sense. I will add it in the `keys` section of mini.pairs to make sure its shows up in which-key.